CIP-dapp-bridge-rework/README.md

```diff
@@ -0,0 +1,1169 @@
+---
+CIP: ????
+Title: Cardano dApp-Wallet Web Bridge
```

Comment:
Therefore we would not really be "reworking" anything (hence we need a new title for this PR) and the CIP would be named to reflect this new serialization method: particularly, naming by the _characteristics_ of your proposed Tx interface rather than any name you would give to e.g. a support library that @Crypto2099 was suggesting at today's meeting.

Reply:
But it's not a serialization method. It's a "dapp bridge with adaptive level of abstraction", which is quite long for a name, I think. Any suggestions?

Reply:
my apologies for not focusing on the part of your proposal that is truly distinctive... what about:
- _dApp-to-Wallet Abstraction Layer_ - if it ends up as a stand-alone method
- _Web-Wallet Bridge - Abstraction Layer for dApps_ - if it becomes an extention to CIP-30 as possible per https://github.com/cardano-foundation/CIPs/pull/623#issuecomment-1830623796